app/assets/stylesheets/welcome.scss

```diff
@@ -46,12 +44,99 @@ input#url_original_url {
 
 .btn {
   border-radius: 25px;
+  background: rgba(209, 85, 233, 0.72);
+  border: #9516c7;
+  padding: 6px;
 }
 
 #text_box {
-  margin-top: 50px;
+  margin-top: 5px;
   border-radius: 25px;
   padding: 6px;
-  border: solid 2px rgba(86, 210, 229, 0.72);
+  border: solid 2px #38d5fb;
   outline: none;
+  padding-right: 20px;
+}
+
+li {
+  list-style: none;
+  color: #38d5fb;
+  font-size: 15px;
+}
+
+a:hover {
+  color: #38d5fb;
+  text-decoration: none;
+}
+
+a {
+  color: white;
+}
+
+.font-icon {
+  color: #ff925c;
+}
+
+// .create-shortened-url {
+//   padding: 50px;
+// }
+
+.url-list {
+  background: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.